src/blocks.tsx

```diff
@@ -0,0 +1,343 @@
+    return (
+        <div aria-label={ariaLabel}>
+            <div
+                className={sprinkles({
+                    display: 'flex',
+                    alignItems: 'center',
+                })}
+            >
+                {image && (
+                    <div
+                        className={sprinkles({
+                            paddingRight: 16,
+                        })}
+                        style={{zIndex: '0'}}
+                    >
+                        {image}
+                    </div>
+                )}
+
+                <Text2 regular color={vars.colors.textSecondary}>
+                    {description}
+                </Text2>
+            </div>
+        </div>
+    );
+};
+
+interface InformationBlockProps {
+    title?: string;
+    description?: string | Array<string>;
+    value: string;
+    secondaryValue?: string;
+    valueColor?: string;
+    'aria-label'?: string;
+}
+
+export const InformationBlock: React.FC<InformationBlockProps> = ({
+    title,
+    description,
+    secondaryValue,
+    value,
+    valueColor = vars.colors.textPrimary,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <Inline space="between" alignItems="flex-end" aria-label={ariaLabel}>
+            <BlockContent title={title} description={description} />
+            <div className={classNames(styles.column, styles.container)}>
+                <Text2 regular color={vars.colors.textSecondary}>
+                    {secondaryValue}
+                </Text2>
+                <Text5 color={valueColor}>{value}</Text5>
+            </div>
+        </Inline>
+    );
+};
+
+interface HighlightedValueBlockProps {
+    tag?: RendersNullableElement<typeof Tag>;
+
+    value: string | Array<string>;
+    text: string | Array<string>;
+
+    secondaryValue?: string;
+
+    title?: string;
+    description?: string | Array<string>;
+
+    valueColor?: string;
+    'aria-label'?: string;
+}
+
+export const HighlightedValueBlock: React.FC<HighlightedValueBlockProps> = ({
+    tag,
+    value,
+    text,
+    secondaryValue,
+    title,
+    description,
+    valueColor = vars.colors.textPrimary,
+    'aria-label': ariaLabel,
+}) => {
+    const [firtValue, secondValue] = value && Array.isArray(value) ? value : [value, ''];
+    const [firtText, secondText] = text && Array.isArray(text) ? text : [text, ''];
+
+    return (
+        <div aria-label={ariaLabel}>
+            {tag && (
+                <div
+                    className={sprinkles({
+                        paddingBottom: 24,
+                    })}
+                >
+                    {tag}
+                </div>
+            )}
+
+            <Stack space={2}>
+                <Inline space={8} alignItems="baseline">
+                    <Text8 color={valueColor}>{firtValue}</Text8>
+                    <Text2 regular color={vars.colors.textSecondary}>
+                        {firtText}
```

Comment:
we are allowing here a secondaryValue without a fistValue, is this correct? 

Maybe we should pass mainValue and secondaryValue as separate props, being both an object providing the value and the text

Something like:

mainHeading={{value: 'Heading', text: 'small text to the right'}}
secondaryHeading={{value: 'Heading2', text: 'small text to the right'}}

This way, we can't have a text without it's 'heading', and we are not using an array when we only have the possibility of two headings.